### DISCUSSION TITLE
Remove unnecessary requirements file

### DISCUSSION THREAD

**Discussion by wigging** [watertap-org/watertap#815] [General]
The repository contains two requirements files. The `requirements.txt` contains

```
# Defer to setup.py contents
-e .
```

and the `requirements-dev.txt` contains

```
black==22.3.0
pre-commit
# Defer to setup.py contents
-e .[dev]
```

Why are there two requirements files instead of just one `requirements.txt`?

**Comment by lbianchi-lbl** [comment]
Thanks (once again) for spotting this. I think this is thankfully pretty clear-cut: one of the files, namely `requirements.txt` is not needed and can (and IMO therefore should) safely be deleted. I don't think this would have any side-effect at all, so if you want to open a PR with this small change, I'd be happy to look at it.

**Reply by wigging** [reply]
I submitted pull request #820 regarding this. So let's continue the discussion on the pull request page.
